scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -0,0 +1,25 @@
+WebPublisherManagerController.$inject = ['$scope', 'api'];
+export function WebPublisherManagerController($scope, api) {
+	$scope.sites = ['content'];
+	$scope.manageSite = false;
+	
+    $scope.toogleCreateSite = function() {
+    	$scope.new = {};
+        $scope.manageSite = !$scope.manageSite;
+    };
+
+    $scope.save = function() {
```

Comment:
This function doesn't need to be on the scope since we don't need a two-way binding here. Just use `this.save` please.